app/src/validators/ats.ts

```diff
@@ -0,0 +1,33 @@
+import Joi from 'joi';
+
+import { validate } from '../middleware/validation';
+
+import { BasicResponse } from '../utils/enums/application';
+
+const addressBody = {
+  '@type': 'AddressResource',
```

Comment:
I'm assuming Joi interprets this as an exact match? But I don't see anything in their documentation about this, if you have it please share. `Joi.string().valid('AddressResource')` would align with what I am seeing.

Reply:
apologies, didn't find any documentation, it just worked. I have changed it to Joi.string().valid('AddressResource'). Thanks for finding it.